### DISCUSSION TITLE
No store cache, and anything else in the session:

### DISCUSSION THREAD

**Discussion by ronyjah** [qutebrowser/qutebrowser#7919] [Q&A]
My HTML javascript section has fetch Options with no-cache and no-store parameters to not save during the session. It works on Firefox and Chrome but I had no success on Qutebrowser. The first time it calls the JS function by clicking on the HTML button, it makes the HTTP request and receives from server the file and on Qutebrowser choose the directory local to save the file. Until here it is perfect but after pressing F5 on the page and clicking on the button again the Qutebrowser does not make the HTTP request anymore, it goes directly to open the box to choose the directory to download to save the file. I read the discussion #6485, delete the directory cache, at moment, is the way to fix the problem? 

        var fetchOptions = {
            method: 'GET',
            cache: 'no-store', // Configura a política de cache para 'no-store'
            'Cache-Control': 'no-cache, no-store', 
        };

        var url = 'http://127.0.0.1:5000/report';

        link.addEventListener('click', function (event) {
            event.preventDefault();
            fetch(url,fetchOptions)
                .then(function (response) {
                    if (response.statusText === "None") {
                        console.log("AUTOBEC")
                        window.location.reload();
                    } else {
                        return response.blob();
                    }
                })

Ubuntu 18.04
installed qutebrowser using apt get-install
Git commit: 
Backend: QtWebEngine (Chromium 56.0.2924.122)

CPython: 3.6.9
Qt: 5.9.5
PyQt: 5.10.1

sip: 4.19.7
colorama: 0.3.7
pypeg2: 2.15
jinja2: 2.10
pygments: 2.2.0
yaml: 3.12
cssutils: no
attr: 17.4.0
PyQt5.QtWebEngineWidgets: yes
PyQt5.QtWebKitWidgets: no
pdf.js: no
sqlite: 3.22.0
QtNetwork SSL: OpenSSL 1.1.1  11 Sep 2018

**Comment by The-Compiler** [comment]
This sounds like it might be a QtWebEngine bug. Do you have a [self-contained, runnable example](http://www.sscce.org/) I could use to report this upstream?
